service/api.go

```diff
@@ -346,6 +345,30 @@ func (as *APIServer) FetchExpansionsForTask(w http.ResponseWriter, r *http.Reque
 		as.LoggedError(w, r, http.StatusNotFound, errors.New("version not found"))
 		return
 	}
+	var u *user.DBUser
+	if t.ActivatedBy == evergreen.StepbackTaskActivator {
+		u, err = user.FindOneById(v.Author)
+	} else if t.ActivatedBy != "" {
+		u, err = user.FindOneById(t.ActivatedBy)
+	}
+	if err != nil {
+		as.LoggedError(w, r, http.StatusInternalServerError, err)
+		return
+	}
+	if u != nil {
```

Comment:
should u being nil be treated as an error?

Reply:
i dont think so because there could be a non-user activated by and we wouldnt want to error every time we get a nil user